api/go.mod

```diff
@@ -0,0 +1,50 @@
+module github.com/openshift/assisted-service/api
+
+go 1.17
+
+require (
+	github.com/openshift/assisted-service v1.0.9
```

Comment:
api module depends on the "root" module?

Reply:
This seems wrong

Reply:
Sorry you are right. strange how it worked. thanks will fix